google-cloud-storage_transfer/samples/acceptance/helper.rb

```diff
@@ -86,6 +105,13 @@ def create_bucket_helper bucket_name
   end
 end
 
+def create_nearline_bucket_helper bucket_name
```

Comment:
nit: Can we use the existing `create_bucket_helper` method and pass the storage_class we need?